docarray/typing/tensor/abstract_tensor.py

```diff
@@ -88,7 +88,7 @@ def __docarray_validate_shape__(cls, t: T, shape: Tuple[Union[int, str]]) -> T:
         :param shape: The shape to validate against.
         :return: The validated tensor.
         """
-        comp_be = t.get_comp_backend()()  # mypy Generics require instantiation
+        comp_be = t.get_comp_backend()  # mypy Generics require instantiation
```

Comment:
```suggestion
        comp_be = t.get_comp_backend()
```

The comment isnt true anymore because it is already instantiated by the method